landing-page/LandingPage.js

```diff
@@ -0,0 +1,52 @@
+import { gql, useQuery } from "@apollo/client";
+import { Box } from "@react-native-material/core";
+
+import SHButton from "../components/SHButton";
+import LoadingPage from "../loading-page/LoadingPage";
+
+const SERVICES_QUERY = gql`
+  {
+    services {
+      edges {
+        node {
+          id
+          name
+          identifier
+          active
+        }
+      }
+    }
+  }
+`;
+
+const servicesList = [];
+
+export default function LandingPage({ navigation }) {
+  const { data, loading } = useQuery(SERVICES_QUERY);
+  if (loading) {
+    return <LoadingPage />;
+  }
```

Comment:
Das abrufen der Daten bitte in ein eigenes File um einfacher Testen zu können. Tests fehlen für diese Component komplett.